pkg/reconciler/pipelinerun/resources/pipelineref.go

```diff
@@ -98,28 +104,32 @@ type LocalPipelineRefResolver struct {
 
 // GetPipeline will resolve a Pipeline from the local cluster using a versioned Tekton client. It will
 // return an error if it can't find an appropriate Pipeline for any reason.
-func (l *LocalPipelineRefResolver) GetPipeline(ctx context.Context, name string) (v1beta1.PipelineObject, error) {
+// TODO (@chuangw6): if we want to set source for in-cluster pipeline, set it here.
```

Comment:
It may be better to link TODO to an issue instead of a person

Reply:
SG! The issue link was already on the next line. I'll remove my name and just leave the link there instead :D. Thanks!